app/assets/styles/styles.css

```diff
@@ -52,6 +52,12 @@ p {
   padding: 0.5em 1em;
 }
 
+.menu__news-name {
+  position: absolute;
```

Comment:
Why does this need to be positioned using absolute?

Reply:
Because it is inside of the `menu` class and all the text it's aligned to the right. I added to use the `left` property.

Reply:
I don't think we need to use an absolute position. There isn't anything special about it that would not let it stay within the flow.

Reply:
You can easily just add `margin-left` on the whole menu (on large screens) so that it starts further left, after the logo. 

In general, we want to use absolute positioning only when it is absolutely needed to pull an element out of the page flow, such as you did for the logo.